resources.nix

```diff
@@ -92,15 +92,9 @@
         name = "evmos";
         version = "v12.1.2";
```

Comment:
Let's also update the version here. Curious why the version went down, do you know anything about that?